client/sites-dashboard/components/sites-search.ts

```diff
@@ -0,0 +1,27 @@
+import Search from '@automattic/search';
+import styled from '@emotion/styled';
+
+export const SitesSearch = styled( Search )`
+	--color-surface: #f6f7f7;
+	border-radius: 4px;
+	overflow: hidden;
+	height: 44px;
+
+	.search-component__icon-search {
+		background: var( --color-surface );
+		height: 100%;
+		fill: currentColor;
+		height: 100%;
+
+		// Flip horizontally
+		transform: scale( -1, 1 );
+
+		padding-right: 8px;
+		padding-left: 5px;
+	}
```

Comment:
This should not be necessary. Ideally, we should be able to pass either:
- a custom icon;
- a class name to modify the icon styling.

I'll create something...

Reply:
https://github.com/Automattic/wp-calypso/compare/add/search-custom-icon-prop?expand=1

Reply:
https://github.com/Automattic/wp-calypso/pull/65561